Makefile

```diff
@@ -18,7 +18,7 @@ DB_PASS ?= kuma # default for ephemeral demo DBs
 target = kuma
 requirements = -r requirements/local.txt
 # set Django settings module if not already set as env var
-export DJANGO_SETTINGS_MODULE ?= kuma.settings.testing
+export DJANGO_SETTINGS_MODULE ?= kuma.settings.prod
```

Comment:
I think it's safest for the default value of the `DJANGO_SETTINGS_MODULE` to remain at `kuma.settings.testing` here (as this `Makefile` is used in multiple contexts), and instead do a `env.DJANGO_SETTINGS_MODULE = 'kuma.settings.prod'` in the `Jenkinsfiles/demo.groovy` file.